src/lib/utils.ts

```diff
@@ -469,3 +469,7 @@ export function binaryToUuid(
     hexValues.slice(10, 16).join(''),
   ].join('-')
 }
+
+export function sourceRangeFileIndex(sourceRange: SourceRange) {
+  return sourceRange[2]
```

Comment:
I guess they don't have to match, but FYI, on the Rust side, we call it the module ID. It's much less ambiguous, IMO.

Reply:
Yeah that's much better, thanks.